pyocd/board/board_ids.py

```diff
@@ -195,24 +195,24 @@ def __init__(self, name, target, binary):
     "1600": BoardInfo(  "Bambino 210",          "lpc4330",          "l1_lpc4330.bin",       ),
     "1605": BoardInfo(  "Bambino 210E",         "lpc4330",          "l1_lpc4330.bin",       ),
     "1900": BoardInfo(  "CY8CKIT-062-WIFI-BT",  "cy8c6xx7",         "l1_cy8c6xx7.bin",      ),
-    "1901": BoardInfo(  "CY8CPROTO-062-4343W",  "cy8c6xxA",         "l1_cy8c6xxa.bin",      ),
+    "1901": BoardInfo(  "CY8CPROTO-062-4343W",  "cy8c6xxa",         "l1_cy8c6xxa.bin",      ),
     "1902": BoardInfo(  "CY8CKIT-062-BLE",      "cy8c6xx7",         "l1_cy8c6xx7.bin",      ),
     "1903": BoardInfo(  "CYW9P62S1-43012EVB-01","cy8c6xx7_s25fs512s", "l1_cy8c6xx7.bin",    ),
     "1904": BoardInfo(  "CY8CPROTO-063-BLE",    "cy8c6xx7_nosmif",  "l1_cy8c6xx7.bin",      ),
-    "1905": BoardInfo(  "CY8CKIT-062-4343W",    "cy8c6xxA",         "l1_cy8c6xxa.bin",      ),
+    "1905": BoardInfo(  "CY8CKIT-062-4343W",    "cy8c6xxa",         "l1_cy8c6xxa.bin",      ),
     "1906": BoardInfo(  "CYW943012P6EVB-01",    "cy8c6xx7",         "l1_cy8c6xx7.bin",      ),
     "1907": BoardInfo(  "CY8CPROTO-064-SB",     "cy8c64xx_cm4_s25hx512t", "l1_cy8c6xx7.bin",),
     "1908": BoardInfo(  "CYW9P62S1-43438EVB-01","cy8c6xx7",         "l1_cy8c6xx7.bin",      ),
-    "1909": BoardInfo(  "CY8CPROTO-062S2-43012","cy8c6xxA",         "l1_cy8c6xxa.bin",      ),
+    "1909": BoardInfo(  "CY8CPROTO-062S2-43012","cy8c6xxa",         "l1_cy8c6xxa.bin",      ),
     "190A": BoardInfo(  "CY8CKIT-064S2-4343W",  "cy8c64xA_cm4",     "l1_cy8c6xxa.bin",      ),
```

Comment:
This one was missed. I agree with @flit. It is better to modify the code to be case-insensitive.